www/webapp/templates/admin/index.html

```diff
@@ -9,22 +9,49 @@
     <div class="d-flex justify-content-between">
         <h1>Courses List</h1>
         <div class="d-flex align-items-center">
-            <a class="btn btn-primary" href="/admin/courses/new">New Course</a>
+            <a class="btn btn-primary" href="/admin/course/new">New Course</a>
         </div>
     </div>
     
     <div class="h-full">
-        {% if not courses %}
-        <div class="d-flex justify-content-center align-items-center h-full">
+        <div id="coursesPlaceholder" 
+            class="d-flex justify-content-center align-items-center h-full">
             <h3>No courses, Please create one</h3>
         </div>
-        {% else %}
-        list of courses
-        {% endif %}
+
+        <div id="courses">
+        </div>
     </div>
 </div>
 <script src="/static/js/utils/authorizer.js"></script>
+<script src="/static/js/admin/index.js"></script>
 <script>
     authorizer([UserTypes.ADMIN, UserTypes.TEACHER]);
+    
```

Comment:
why do you not put in index.js?

Reply:
I'm thinking on visibility, to make it simpler to read the permission scope it will be trickier if you need to read the external file.

But not sure if there's a better way or if it's even a real problem